includes/data/post.json

```diff
@@ -0,0 +1,27 @@
+[
```

Comment:
this code is not part of the hero image.

Reply:
done